cluster/gce/gci/configure-helper.sh

```diff
@@ -2076,6 +2076,42 @@ function start-fluentd-resource-update {
   wait-for-apiserver-and-update-fluentd &
 }
 
+# VolumeSnapshot CRDs are installed by cluster addon manager, which may
+# not be available at this point. Run this as a background process.
+function wait-for-apiserver-and-update-volumesnapshot {
+  # Wait until volumesnapshot CRDs is in place.
+  until kubectl get volumesnapshotclasses.snapshot.storage.k8s.io
```

Comment:
Should we log some messages to ease debugging?

Reply:
sure